src/FastTags/LexerTypes.hs

```diff
@@ -77,8 +78,13 @@ mkAlexInput s trackPrefixes = AlexInput
     utf8BOM' = "\xFEFF"
 
 mkSrcPos :: FilePath -> AlexInput -> SrcPos
-mkSrcPos filename (AlexInput {aiLine, aiPrefix}) =
-    SrcPos { posFile = filename, posLine = aiLine, posPrefix = aiPrefix }
+mkSrcPos filename (AlexInput {aiLine, aiPrefix, aiAbsPos}) =
+    SrcPos { posFile = filename
+           , posLine = aiLine
+           , posOffset = Offset aiAbsPos
+           , posPrefix = aiPrefix
+           , posText = aiPrefix <> (fromString $ show $ unLine aiLine)
```

Comment:
this is just a placeholder...

Reply:
we have the `aiInput` and the location of the match, so it should be a case of taking that position to the next newline.